devtools/armdevtools.py

```diff
@@ -8,6 +8,7 @@
 import armui
 import log
 import database
+import qacheck
```

Comment:
Don't forget to add this to the requirements.txt

Reply:
updated requirements with flake8 to support the devtools